source/manual/how-to-test-with-assitive-technology.html.html.md

```diff
@@ -0,0 +1,38 @@
+---
+owner_slack: "#govuk-frontenders"
+title: How to test with Assistiv Labs
+section: Frontend
+layout: manual_layout
+parent: "/manual.html"
+---
+
+GOV.UK has to meet government [accessibility requirements](https://www.gov.uk/service-manual/helping-people-to-use-your-service/making-your-service-accessible-an-introduction), digital services must work on the most commonly used assistive technologies - including screen magnifiers, screen readers and speech recognition tools
+
+[Assistiv Labs](https://assistivlabs.com/) remotely connects you to real assistive technologies, like NVDA, JAWS, and Windows High Contrast Mode, using any modern web browser.
+
+## Get setup
```

Comment:
Tiny nitpick - here and I think in one other instance you say "setup" without a space which is a noun, you should be using "set up" as a verb.

Reply:
Huh, did not know this - updated